### PR TITLE
Implemented ChainerX version of Clipped ReLU forward

### PR DESCRIPTION
**Pull request by Harshan01**
With reference to #6622 , ChainerX version of Clipped ReLU forward is implemented using 'chainerx.minimum' implemented with respect to #6477

### CODE REVIEW

File: chainer/functions/activation/clipped_relu.py [hunk 1]
```diff
@@ -36,6 +37,11 @@ def check_type_forward(self, in_types):
         x_type = in_types[0]
         type_check.expect(x_type.dtype.kind == 'f')
 
+    def forward_chainerx(self, inputs):
+        self.retain_inputs((0,))
+        x, = inputs
+        return chainerx.minimum(chainerx.maximum(0, x), self.cap)
```
Comment (asi1024) [inline chainer/functions/activation/clipped_relu.py]:
```suggestion
        return chainerx.minimum(chainerx.maximum(0, x), self.cap),
```

File: chainer/functions/activation/clipped_relu.py [hunk 2]
```diff
@@ -36,6 +37,11 @@ def check_type_forward(self, in_types):
         x_type = in_types[0]
         type_check.expect(x_type.dtype.kind == 'f')
 
+    def forward_chainerx(self, inputs):
+        self.retain_inputs((0,))
```
Comment (asi1024) [inline chainer/functions/activation/clipped_relu.py]: This line is not needed.

**Comment by Harshan01** [comment]
@asi1024 Can you check this out?

**Comment by asi1024** [comment]
Jenkins, test this please.

**Comment by asi1024** [comment]
Jenkins failure is unrelated. LGTM!

**Pull request merged**